src/app/components/FatalErrorHandler/index.tsx

```diff
@@ -61,13 +63,7 @@ export function FatalErrorHandler(props: Props) {
           }}
           pad={{ horizontal: 'small', vertical: 'xsmall' }}
         >
-          <Box>
-            <Text weight="bold" data-testid="fatalerror-message">
-              {fatalError.message}
-            </Text>
-          </Box>
-          <pre>{fatalError.sagaStack}</pre>
-          <pre>{fatalError.stack}</pre>
```

Comment:
> interesting that we have two different preparations of the text: one plaintext for copying, one constructed in HTML`<pre>`s and now `<br>`s for display

Good point - deduplicated it now. `<Box><Text weight="bold">message` looked nearly the same as `<pre>`, so I'd rather reduce complexity